### PR TITLE
[de]: Translate missing keys

### PR DESCRIPTION
**Pull request by julianstark999**
Translated missing keys for german translation

### CODE REVIEW

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
@WhereIsLucas, @sotten, can you check this PR?

File: locales/de/php-inline.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "ipv4": "Das muss eine gültige IPv4-Adresse sein.",
     "ipv6": "Das muss eine gültige IPv6-Adresse sein.",
     "json": "Das muss ein gültiger JSON-String sein.",
-    "lowercase": "This field must be lowercase.",
+    "lowercase": "Das muss in kleinbuchstaben sein.",
```
Comment (sotten) [inline locales/de/php-inline.json:54]: Kleinbuchstaben

Comment (julianstark999) [reply locales/de/php-inline.json]: Fixed my typo

**Review by sotten — APPROVED** [review]

**Review by andrey-helldar — APPROVED** [review]
Danke

**Pull request merged**